src/peft/tuners/xlora/model.py

```diff
@@ -135,7 +200,40 @@ def __init__(
         model: nn.Module,
         config: Union[dict[str, XLoraConfig], XLoraConfig],
         adapter_name: str,
+        torch_device: Optional[str] = None,
+        ephemeral_gpu_offload: bool = False,
+        autocast_adapter_dtype: bool = True,
+        **kwargs,
     ) -> None:
+        """
+        Create a new X-LoRA model
+
+        Args:
+            model (`nn.Module`):
+                Base model to apply X-LoRA to.
+            config: ([`XLoraConfig`]):
+                X-LoRA configuration object.
+            adapter_name: (`str`):
+                Adapter name for the X-LoRA adapter.
+            torch_device (`str`, *optional*, defaults to None):
+                (For loading the LoRA adapters) The device to load the adapter on. If `None`, the device will be
+                inferred.
+            autocast_adapter_dtype (`bool`, *optional*, defaults to `True`):
+                (For loading the LoRA adapters) Whether to autocast the adapter dtype. Defaults to `True`. Right now,
+                this will only cast adapter weights using float16 and bfloat16 to float32, as this is typically
+                required for stable training, and only affect select PEFT tuners.
+            ephemeral_gpu_offload (`bool`, *optional*, defaults to `False`):
+                (For loading the LoRA adapters) Whether to use ephemeral GPU offloading for partially loaded modules.
+                Defaults to `False`.
+            autocast_adapter_dtype (`bool`, *optional*, defaults to `True`):
```

Comment:
This argument is duplicated in the docstring.

Reply:
Thanks, fixed!